### ISSUE TITLE
64-bit dll is not 64-bit

### ISSUE THREAD

**Issue by Sequiturian** [sebity/plOpenGL#1]
The 64-bit swi-prolog on Windows cannot load the plOpenGL.dll because it has the wrong machine header (x86 and not amd64 or x86_64).
The build must not work properly for 64-bit or the pack was put together wrongly.
Both dll's were checked: i386-win32 and x64-win64 folders.

**Comment by sebity** [comment]
I have released an updated pack on swi-prolog.org, plOpenGL v0.6.2 which has been tested on Windows 7 using SWI-Prolog v7.1.28 for x64-win64.

Yes, the problem was partly to do with the pack, as at the time when I created the pack I didn't realise that the Windows 7 was running a 32bit version of SWI-Prolog when I created the dll.  I only noticed today when I tried to install the pack, on the same netbook that created the dll a year ago. I develop on Linux and was asked a year ago to create a pack with the dll's as the user had difficulty compiling with mingw on Windows.
